Makefile

```diff
@@ -5,6 +5,16 @@ PRODUCTION_URL="https://docs.mongodb.com"
 STAGING_BUCKET=docs-mongodb-org-stg
 PRODUCTION_BUCKET=docs-mongodb-org-prd
 
+ifeq ($(ENV), 'dotcom_stg')
```

Comment:
Do you think we should avoid calling this "ENV" to avoid conflict with the unix `env` command? Maybe something like docsenv or publishenv?